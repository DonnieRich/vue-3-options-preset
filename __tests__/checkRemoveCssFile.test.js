const { vol } = require("memfs");

jest.mock("fs/promises");

const { removeCssFile } = require("../src/bin/removeCssFile");

describe(removeCssFile, () => {
    beforeEach(() => {
        vol.reset();
    });

    it("It should pass if the file style.css has been found and removed", async () => {

        vol.fromJSON(
            {
                "/src/style.css": "my style"
            },
            "/"
        );

        await removeCssFile("/src");
        expect(vol.toJSON()).toMatchSnapshot();
    });
});