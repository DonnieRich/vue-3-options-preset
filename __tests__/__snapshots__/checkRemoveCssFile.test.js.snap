// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`removeCssFile It should pass if the file style.css has been found and removed 1`] = `
{
  "/src": null,
}
`;
